cli/build.rs

```diff
@@ -1,9 +1,17 @@
-use std::process::Command;
+use clap::{CommandFactory, ValueEnum};
+use clap_complete::{generate_to, Shell};
+use clap_mangen::Man;
 
-fn main() {
-    println!("cargo:rerun-if-env-changed=TYPST_VERSION");
-    if option_env!("TYPST_VERSION").is_some() {
-        return;
+use std::{
+    env,
+    fs::{create_dir_all, File},
+    path::Path,
+    process::Command,
+};
```

Comment:
can you split this up into one line per module and move it above the third party imports?